build.sbt

```diff
@@ -24,8 +24,8 @@ val appName = "business-registration"
 
 lazy val scoverageSettings = Seq(
   ScoverageKeys.coverageExcludedPackages := "<empty>;controllers.beta.*;Reverse.*;model.*;config.*;.*(AuthService|BuildInfo|Routes).*",
-  //ScoverageKeys.coverageMinimum := 80,
-  ScoverageKeys.coverageFailOnMinimum := false,
+  ScoverageKeys.coverageMinimumStmtTotal := 80,
+  ScoverageKeys.coverageFailOnMinimum := true,
```

Comment:
Do we want this to fail on minimum I don't believe any of the others are doing

Reply:
Personally, I feel like we should be going forward. That way if anyone adds something that takes the coverage below minimum it will fail the build and not create a release. So then we can investigate why the coverage has dropped - as it could be a regression somewhere or could be some new code has missing scenarios from tests 🤷‍♂️

Reply:
Feels like it's gonna be a bit hit or miss across the services on whether people will care though. That will be determined by whether we turned on the fail or not, would be good if we could bring them all to the >80% mark as a benchmark for failure and enable on all rather than the odd few.

But that's just my chat, I agree we shouldn't drop below that now so it's a merge from me :)